validators_test.go

```diff
@@ -175,6 +175,22 @@ func TestValidators__isAlphanumeric(t *testing.T) {
 	}
 }
 
+func TestValidators__isAlphanumericExamples(t *testing.T) {
+	v := validator{}
+
+	validCases := []string{`|`}
```

Comment:
I'm curious, is `|` alphanumeric (letters and numbers)?

Reply:
`|` is `0x7C` [in ASCII](https://en.wikipedia.org/wiki/ASCII#Character_set) which is allowed by Nacha.